### DISCUSSION TITLE
四月更新 - 3

### DISCUSSION THREAD

**Discussion by Richasy** [Clean-Reader/CleanReader.Desktop#26] [Announcements]
## 🚀 功能

- 支持显示阅读记录详情 @Richasy (#23)
- 书架上的书籍支持另存为 @Richasy (#22)

## 🐛 Bug 修复

- 修复网络下载转换过程中失败的问题 @Richasy (#25)
- 修复分栏阅读设置未生效的问题 @Richasy (#21)

**Comment by ltxman** [comment]
试了几次还是不行，下载进度结束就显示这个
![image](https://user-images.githubusercontent.com/62414615/163559898-7b4d52d1-b789-427c-979d-0acec40d4240.png)

**Reply by Richasy** [reply]
你是不是开了网络代理什么的？

**Reply by ltxman** [reply]
没有呀，哦对，可能是校园网的问题，我明天用流量试试，谢谢大佬！

**Reply by Richasy** [reply]
不客气，你也可以尝试其它的书源，我扩展了一些，在这个仓库： [CleanReader.Sources](https://github.com/Clean-Reader/CleanReader.Sources)，你可以下载后使用单独的书源搜索书籍并下载（因为实测下来，你遇到的这个情况可能会在部分书源或者部分书籍上发生，这时候切换一下就好）

**Reply by ltxman** [reply]
好！感谢！
